lib/new-admin/supervisor.js

```diff
@@ -1,35 +1,57 @@
 const xmlrpc = require('xmlrpc')
+const logger = require('../logger')
+const { promisify } = require('util')
 
 function getAllProcessInfo () {
   const convertStates = (state) => {
+    // From http://supervisord.org/subprocess.html#process-states
     switch (state) {
-      case 'EXITED':
-        return 'STOPPED'
-      case 'STOPPING':
+      case 'STOPPED':
         return 'STOPPED'
       case 'STARTING':
         return 'RUNNING'
+      case 'RUNNING':
+        return 'RUNNING'
       case 'BACKOFF':
         return 'FATAL'
+      case 'STOPPING':
+        return 'STOPPED'
+      case 'EXITED':
+        return 'STOPPED'
+      case 'UNKNOWN':
+        return 'FATAL'
       default:
-        return state
+        return 'FATAL'
```

Comment:
Should also log that we've seen an unsupported state, in case of ``default``.

Reply:
Resolved in https://github.com/RafaelTaranto/lamassu-server/pull/1/commits/ef399b0987842b56ceaf31347a40b4ecfa356e69.